### PR TITLE
add crows_pairs

### PR DESCRIPTION
**Pull request by patil-suraj**
This PR adds CrowS-Pairs datasets.

More info:
https://github.com/nyu-mll/crows-pairs/
https://arxiv.org/pdf/2010.00133.pdf

### CODE REVIEW

File: datasets/crows_pairs/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
```
Comment (patil-suraj) [inline datasets/crows_pairs/README.md:19]: Bit unsure about the task tag. The dataset is created to measure the degree to which U.S. stereotypical biases present in the MLMs, so using `language-modeling` tag here. Wdyt @lhoestq @yjernite ?

File: datasets/crows_pairs/crows_pairs.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            ),
+            supervised_keys=None,
+            citation=_CITATION,
+            homepage="https://github.com/nyu-mll/crows-pairs",
+        )
+
+    def _split_generators(self, dl_manager):
+        filepath = dl_manager.download(_URL)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TEST, gen_kwargs={"filepath": filepath}),
```
Comment (patil-suraj) [inline datasets/crows_pairs/crows_pairs.py:96]: The data is only provided for evaluation purpose, so just returning the `TEST` split.

File: datasets/crows_pairs/README.md (hunk 1 above)
Comment (lhoestq) [reply datasets/crows_pairs/README.md:19]: maybe add another task ?
```
task_categories:
- sequence-modeling
task_ids:
- language-modeling
- sequence-modeling-other-social-biais-detection
```

**Review by lhoestq — APPROVED** [review]
Really nice ! thanks for adding this one.

feel free to merge if you have no more changes to do

File: datasets/crows_pairs/README.md (hunk 1 above)
Comment (yjernite) [reply datasets/crows_pairs/README.md:19]: @patil-suraj there's also a ``text-scoring` task category which would be useful there, you can add `other-bias-evaluation` as a specific task

**Comment by lhoestq** [comment]
looks good now :) wdyt @yjernite ?

**Comment by patil-suraj** [comment]
Looks good to merge for me, can edit the dataset card later if required. Merging

**Pull request merged**